CONTRIBUTING.md

```diff
@@ -22,9 +22,41 @@ Preferred Package Structure
     * Class names should have "Entity" suffix to differentiate them from their DTO counterparts
 * security
   * authentication, authorization, RBAC, etc.
+-------------
+# Build
+
+Requirements to build the repository are:
+- Gradle 7.4+
```

Comment:
Is this correct?

Reply:
Should we even be specifying? The recommendation is to use the gradlew scripts which will import the necessary gradle version automatically. I believe it is currently Gradle 8.3

Reply:
Good point. I'm fine with removing the version then :)

Reply:
Updated